iree/task/worker.h

```diff
@@ -117,7 +117,12 @@ typedef struct iree_task_worker_t {
   // TODO(benvanik): test on 32-bit platforms; I'm pretty sure we'll always be
   // past the iree_hardware_constructive_interference_size given the bulk of
   // stuff above, but it'd be nice to guarantee it.
-  uint8_t _padding[8];
+  // uint8_t _padding[8];
```

Comment:
Intended to keep commented out? What about just removing this and its comment?

Reply:
will add a comment clarifying when we may need to add it back (where padding is inserted in this struct is load-bearing, so having a placeholder for "<< YOU CAN ADD PADDING HERE AND HERE ONLY >>" is nice)